README.md

```diff
@@ -271,42 +272,49 @@ The following additional software packages must be installed on the operating sy
 
 ### Step 1 - Download the Source
 
-You can choose between two options to get the source code. Either via the checkout of a Git repository or the download of a release archive.
+You can choose between two options to get the source code:
 
+1. [Download the Release Archive](#download-the-latest-release)
+1. [Clone the Repository](#clone-the-repository)
 
-> **Warning**
->
-> A branch is mandatory because it is used for the build version and the virtual machine name. It does not matter if it is based on the HEAD or a release tag.
+   > **Warning**
```

Comment:
This warning exists twice. Either we remove one of them or move this one under the code example.